CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
 
+* Bumps Instabug Android SDK to v11.6.0
```

Comment:
@TheBuggedYRN Do you think we should also add an item for fixing APM network data not being sent on Android?